### PR TITLE
community/emacs upgrade version to 26.3

### PR DESCRIPTION
**Pull request by stef**
minor version upgrade only.

### CODE REVIEW

**Comment by maxice8** [comment]
Please use the correct format  for  updating packages

`<repo>/<package>: upgrade to <pkgver>`

**Comment by stef** [comment]
the CI builders seem to have problems with this, however on my private builders this builds fine. we had the same problem also with the previous version.

**Comment by stef** [comment]
btw i don't see that "bumping" would be incorrect: https://github.com/alpinelinux/aports/search?q=bumped&type=Commits

**Comment by stef** [comment]
damn. i forgot the checksum

**Comment by Cogitri** [comment]
Fails on my machine with the same error as on CI

**Comment by stef** [comment]
which one? on travis-ci the problem is, that ftp is blocked by some firewall rule, on drone io some other security mechanism gets in the way. it's a problem with the ci security settings.

File: community/emacs/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: stef <l0ls0fo2i@ctrlc.hu>
 # Contributor: Timo Teräs <timo.teras@iki.fi>
 pkgname=emacs
-pkgver=26.2
+pkgver=26.3
 pkgrel=2
```
Comment (Cogitri) [inline community/emacs/APKBUILD]: Argh, noticed it too late, but this should've been reset to 0.

Comment (stef) [reply community/emacs/APKBUILD]: hrmpf! sorry! i'll try to not forget in the future.

Comment (Cogitri) [reply community/emacs/APKBUILD]: No worries, I should've caught that.

**Pull request closed without merging**